test/conftest.py

```diff
@@ -5,7 +5,9 @@
 from asv import config
 from asv import repo
 from .test_workflow import generate_basic_conf
-
+import shutil
+from os.path import abspath, dirname
```

Comment:
The imports from Python's standard library should come first.

Reply:
Let me fix them